docs/pages/reference/config.mdx

```diff
@@ -770,12 +770,18 @@ app_service:
       - name: "os"
         command: ["/usr/bin/uname"]
         period: "5s"
-      # Optional simple rewriting of Location header
-      # Rewrite the "Location" header on redirect responses replacing the
-      # host with the public address of this application.
-      # redirect:
-      #   - "localhost"
-      #   - "jenkins.internal.dev"
+      # Optional list of rewrite rules to apply to requests and responses
+      # rewrite:
+        # Optional simple rewriting of Location header
+        # Rewrite the "Location" header on redirect responses replacing the
+        # host with the public address of this application.
+        # redirect:
+        #   - "localhost"
+        #   - "jenkins.internal.dev"
+        # Optional list of extra headers to inject in to requests.
+        # headers:
```

Comment:
Since the whole block is commented out, it's a bit challenging to understand what is the comment and what is the actual field name. I don't know what a good solution would be here, maybe do not comment out actual fields/values  i.e. just show them as they would appear in the config?

Reply:
[Resolved](https://github.com/gravitational/teleport/pull/17496#discussion_r1008377532).